### ISSUE TITLE
[DSOC Drama] Murder By Midnight + Introducing SUTD Drama

### ISSUE THREAD

**Issue by marcelprasetyo** [OpenSUTD/Discovery-Week-Working-Title-#23]
**Session Name**

Murder By Midnight

**Abstract**

Showcase of a short detective comedy to show how acting looks like, followed by introductory session to drama, outlining the goals of the club, perhaps some icebreakers for a smaller audience. A presentation of previous performances that have been done (including All My Sons tragedy), and clips of training sessions.

**Student Organisation organising this session**

DSOC Drama Club

**Session intent (what to expect)**

- be lightly _entertained_
- see for yourself what we do at DSOC _Drama_
- find out if _acting_ , _directing_ , _productions crew_ , _playwriting_ or _props-making_ is your thing

**Session details**

Date : Thursday 30th May 2019, 730 PM - 930 PM
Timing requirements : after dinner and not too late
Ideal Venue : LT5
Venue requirements : a venue that allows some space for small-cast acting and a projector with screen to show a clip of past performances

We expect 30 people to come.

**Speakers/Workshop facilitators**

3 Actors from Drama, all with experience acting before.
Drama members who are able to attend.
Drama exco will be there.

**Logistical details**

We would require about $50 to purchase materials for our set and props that we will use in our play. Props include fake guns, hotel trolley decorations, costumes and simple backdrop. We would also like to get some light refreshments for the freshies during our introductory session.

It would be great to have a logistics trolley as well so that we can make it into a hotel service trolley.

marcelmarheave@gmail.com

**Comment by Fishbiscuit** [comment]
okay i don't think we can source for you a logistics trolley, but I could try and ask someone who can? would that be alright

**Comment by marcelprasetyo** [comment]
> okay i don't think we can source for you a logistics trolley, but I could try and ask someone who can? would that be alright

ah yea that would be fine thanks !

**Comment by Fishbiscuit** [comment]
hello, just to let you know that there's a housing briefing scheduled for 23 May on 1900 onwards, it's a pretty important briefing so freshmores will be expected to attend that too. would you like to change the timing/date? otherwise we could try work out something to key details aren't missed

**Comment by marcelprasetyo** [comment]
Oh thanks for the info. I think Wednesday  22 may 1930 would be okay. does that collide with anything?

**Comment by Fishbiscuit** [comment]
it's right after the human library which is expected to overrun. still possible if you would like to! Also, I just had a chat with my staff liaison and she said that she doesn't have access to certain venues. I would have to trouble you to initiate a booking yourself, would that be fine?

**Comment by marcelprasetyo** [comment]
Yeah that would be fine. By 'certain venues' you mean including LT5 or are you not sure yet? If there's no choice it would be fine but I do not know how to book venues.

**Comment by marcelprasetyo** [comment]
Hi um I think the original date 30 May is okay? Because I didn't realise you were referring to 23 May when you say it clashes originally, which is the first week, and we would like to have it on the second week 30 May, and it says 30 May on the Issue above. Also still confirming with the actors whether they can make it on 30 May., give me a little time.

nb our event is scheduled on 30 may on the calendar hahah

**Comment by tlkh** [comment]
https://discover.sutd.dev/events.html#dsoc-drama

Please get back to us if you want any details changed.

**Comment by marcelprasetyo** [comment]
btw how do I book the venue again?

**Comment by marcelprasetyo** [comment]
> https://discover.sutd.dev/events.html#dsoc-drama
> 
> Please get back to us if you want any details changed.

Hi guys can i change the below part from the description

Showcase of a short detective comedy to show how acting looks like, followed by introductory session to drama, outlining the goals of the club, perhaps some icebreakers for a smaller audience. 

into

Showcase of a short detective comedy (~10 minutes, popcorns provided) to show how acting looks like. (Afterwards you can leave to catch up with other Discovery sessions.) The show is then followed by introductory session to drama which includes a short presentation that outlines the goals of the club, and fun drama-related icebreakers / warm-ups.

And finally if you could help me edit the Timing from 730 - 930 to become:

> 730 - 745 (show) and 745 - 930 (intro session)
,properly formatted.

Adding these things would be great. This does not omit the rest of the description. Thanks a lot!

**Comment by tlkh** [comment]
Ok I have made the changes.

Also if you have not yet settled the venue, @Fishbiscuit to advise

**Comment by Fishbiscuit** [comment]
okay booking has been made for LT5 - you've got to take our staff liaison from OSA for this!

**Comment by marcelprasetyo** [comment]
thanks a lot!
